src/D2L.CodeStyle.Analyzers.Test/D2L.CodeStyle.Analyzers.Test.csproj

```diff
@@ -0,0 +1,134 @@
+﻿<?xml version="1.0" encoding="utf-8"?>

```

Comment:
This (and friends) into `test/`? (and root namespace that matches what is being tested. Also `UnitTest` vs. `IntegrationTest`+categories but I'm not sure we need to cargo-cult that hard (maybe give a look at MS repos)